server/server.js

```diff
@@ -1,4 +1,5 @@
 require('dotenv').config();
+const { default: axios } = require('axios');
```

Comment:
Any reason for the destructuring on axios? I don't think I've ever seen this.

Reply:
Hum, I guess it got automatically added when I wrote the Axios request. I don't remember adding it and I have never seen that either.